yarn-project/aztec-node/src/actions/fast-sync.ts

```diff
@@ -0,0 +1,161 @@
+import { ARCHIVER_DB_VERSION, ARCHIVER_STORE_NAME, type ArchiverDataStore } from '@aztec/archiver';
+import { INITIAL_L2_BLOCK_NUM } from '@aztec/constants';
+import type { L1ContractAddresses, ViemPublicClient } from '@aztec/ethereum';
+import type { Logger } from '@aztec/foundation/log';
+import { DATABASE_VERSION_FILE_NAME } from '@aztec/stdlib/database-version';
+import { type FileStore, createFileStore } from '@aztec/stdlib/file-store';
+import {
+  type SnapshotMetadata,
+  type SnapshotsIndexMetadata,
+  downloadSnapshot,
+  getLatestSnapshotMetadata,
+  makeSnapshotLocalPaths,
+} from '@aztec/stdlib/snapshots';
+import { WORLD_STATE_DB_VERSION, WORLD_STATE_DIR } from '@aztec/world-state';
+
+import { mkdtemp, rename, rm } from 'fs/promises';
+import { join } from 'path';
+
+import type { AztecNodeConfig } from '../aztec-node/config.js';
+
+// Half day worth of L1 blocks
+const MIN_L1_BLOCKS_TO_TRIGGER_REPLACE = 86400 / 2 / 12;
+
+export async function tryFastSync(
```

Comment:
This seems to be uncalled?

Reply:
Should've said it was WIP. It's now called on the node's `createAndSync`, right before initializing the archiver..